element_calcium_imaging/imaging.py

```diff
@@ -494,18 +495,19 @@ def make(self, key):
 
             # -- summary images --
             field_keys = {}
-            for field_key, _ in (Processing.Field * Processing.MemberField
-                              * scan.ScanInfo.Field & key).fetch('KEY', 'field_z', order_by='field_z'):
+            for field_key, _ in zip(*(
+                    Processing.Field * Processing.MemberField
+                    * scan.ScanInfo.Field & key).fetch('KEY', 'field_z', order_by='field_z')):
```

Comment:
Why do we need to fetch `field_z` here if it is not used? `order_by` does not work if not fetching?

Reply:
yes, a DataJoint 0.13.* bug